website_legal_page/views/website_legal.xml

```diff
@@ -48,6 +48,11 @@
               </p>
             </div>
             <div>
+              <t t-if="res_company.company_registry">
```

Comment:
I agree w/ @Yajo: should be conditional. Option: add a flag on website panel "Show company registry".

Reply:
Maybe in the res.company form view, along with the `company_registry` field, a checkbox: "Display in website legal advice page"